src/main/kotlin/net/lumalyte/lg/infrastructure/services/apollo/GuildTeamService.kt

```
@@ -112,30 +110,62 @@ class GuildTeamService(
+
+            val leaderColor = parseColor(config.getString("apollo.teams.colors.leader", "255,215,0"))
+            val officerColor = parseColor(config.getString("apollo.teams.colors.officer", "0,150,255"))
+            val memberColor = parseColor(config.getString("apollo.teams.colors.member", "0,255,0"))
+
+            cachedMarkerColors[playerId] = when {
+                rank == null -> memberColor
+                rank.priority >= 90 -> leaderColor
+                rank.priority >= 50 -> officerColor
+                else -> memberColor
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)